### PR TITLE
Some fixes to the 8.4 migration guide

### PR DESCRIPTION
**Pull request by Girgias**

### CODE REVIEW

**Comment by Girgias** [comment]
@jimwins do you know why it is not picking up the DocBook 5.2 DTD?

**Comment by Girgias** [comment]
I remember ... there is no DTD for 5.2, we need to set-up some sort of RelaxNG check rather than DTD...

File: appendices/migration84/incompatible.xml [hunk 1]
```diff
@@ -459,9 +459,9 @@
 
   <simpara>
    Some DOM methods previously returned &false; or a
-   <constant>PHP_ERR</constant> <exceptionname>DOMException</exceptionname>
+   <constant>DOM_PHP_ERR</constant> <exceptionname>DOMException</exceptionname>
```
Comment (TimWolla) [inline appendices/migration84/incompatible.xml:462]:
```suggestion
   <exceptionname>DOMException</exceptionname> with code <constant>DOM_PHP_ERR</constant>
```

And likewise below? This avoids two identifiers after each other which renders badly.

**Review by TimWolla — COMMENTED** [review]
While at it, please add the missing word "function" here:

https://github.com/php/doc-en/blob/4309102ef25eba1a9aa5a87d24e679286b7927e9/appendices/migration84/deprecated.xml#L94

---------------

LGTM other than these two remarks.

File: appendices/migration84/other-changes.xml [hunk 2]
```diff
@@ -716,7 +727,7 @@
     Improved the performance of <classname>\Random\Randomizer</classname>,
     with a specific focus on the
     <methodname>\Random\Randomizer::getBytes</methodname>,
-    and <methodname>getBytesFromString()</methodname> methods.
+    and <methodname>\Random\Randomizer::getBytesFromString</methodname> methods.
```
Comment (TimWolla) [inline appendices/migration84/other-changes.xml:730]: These should not have the leading backslash. Also affects:

https://github.com/php/doc-en/blob/4309102ef25eba1a9aa5a87d24e679286b7927e9/appendices/migration84/deprecated.xml#L340

File: appendices/migration84/other-changes.xml [hunk 3]
```diff
@@ -716,7 +727,7 @@
     Improved the performance of <classname>\Random\Randomizer</classname>,
```
Comment (TimWolla) [inline appendices/migration84/other-changes.xml:727]: Same here.

**Review by TimWolla — APPROVED** [review]

**Pull request merged**